docs/_posts/2016-01-11-admin-manual.md

```diff
@@ -173,10 +173,7 @@ The configuration files are:
 | `submission.properties`   | properties to set the submission environment.              |
 
 ## Security Environment
-MISO can use either LDAP or JDBC as an authentication mechanism. The mechanism
-is set in both `/storage/miso/security.properties` and the
-`$CATALINA_HOME/bin/setenv.sh` or `/etc/default/tomcat8` files and both must be
-the same.
+MISO can use either LDAP or JDBC as an authentication mechanism.
```

Comment:
Please indicate in which file the authentication property must be set, as the lines below reference setting the security method.